src/scripts/background.js

```diff
@@ -896,9 +904,7 @@ TogglButton = {
       clientName = 0,
       i,
       validate = function (item) {
-        return item.indexOf("</option>") !== -1 &&
-          !!item &&
-          ((item.match(/\/option/g) || []).length > 1 || item.length > 1);
+        return item.indexOf("project-row") !== -1 && !!item;
```

Comment:
if `item` is falsey, unless it is exactly `""` then `item.indexOf()` will throw before `!!item` is validated. if `item` is exactly `""`, then there's no need to check if `!!item`.